cvrf2csaf/section_handlers/product_tree.py

```diff
@@ -4,7 +4,22 @@
 
 class ProductTree(SectionHandler):
     """ Responsible for converting the ProductTree section """
-    
+    branch_type_mapping = {
+        "Vendor": 'vendor',
+        "Product Family": 'product_family',
+        "Product Name": 'product_name',
+        "Product Version": 'product_version',
+        "Patch Level": 'patch_level',
+        "Service Pack": 'service_pack',
+        "Architecture": 'architecture',
+        "Language": 'language',
+        "Legacy": 'legacy',
+        "Specification": 'specification',
+        "Host Name": 'host_name',
+        "Realm": 'N/A',
+        "Resource": 'N/A',
```

Comment:
Hi @tschmidtb51 

1) what to do with these 2 values: `Realm` and `Resource`? They are valid in the input, but have no mapping to the output

2) In general, do you agree with creating these mapping dicts for string values? I like to be explicit and I find this more readable than having .lower() somewhere deeper in the code. Furthermore, sometimes these transformations (like str.lower), aren't enough.

Reply:
there is a conversion rule, Thomas will update this here.

Reply:
**Conversion rule**: Convert them into a `product_name` and add a warning that those types do not exist in CSAF.

Reply:
Done, see:

https://github.com/csaf-tools/CVRF-CSAF-Converter/pull/54/commits/6f543cd05603c27172804e03e6c21546e9ec35a9

Reply:
Btw I haven't seen these types in any example so far

Reply:
BTW: This is now also part of the [editor revision](https://github.com/oasis-tcs/csaf/blob/editor-revision-2022-02-23/csaf_2.0/prose/csaf-v2-editor-draft.md#915-conformance-clause-5-cvrf-csaf-converter).